htdocs/install/mysql/migration/19.0.0-20.0.0.sql

```diff
@@ -205,4 +203,7 @@ ALTER TABLE llx_c_tva ADD COLUMN type_vat smallint NOT NULL DEFAULT 0 AFTER fk_p
 
 ALTER TABLE llx_categorie ADD COLUMN position integer DEFAULT 0 AFTER color;
 
-ALTER TABLE llx_product DROP COLUMN onportal; 
+ALTER TABLE llx_product DROP COLUMN onportal;
+
+RENAME TABLE llx_commande_fournisseur_dispatch_extrafields TO llx_receptiondet_batch_extrafields;
```

Comment:
To rename a table, you must use "ALTER TABLE ... RENAME ..."  (se example at top of the file).

also you must update the trigger used for pgslq:

-- VPGSL8.2 CREATE TRIGGER update_customer_modtime BEFORE UPDATE ON llx_receptiondet_batch FOR EACH ROW EXECUTE PROCEDURE update_modified_column_tms();

Reply:
hello @eldy it seems that RENAME TABLE is also good (https://dev.mysql.com/doc/refman/8.0/en/rename-table.html and https://popsql.com/learn-sql/mysql/how-to-rename-a-table-in-mysql) but maybe you have a good reason ti not use it ?

Reply:
trigger updated here if i understood what you said : https://github.com/Dolibarr/dolibarr/pull/27528/commits/933a36cf1e257156b4b1448b349ee8df8521184d

Reply:
also change the way to rename table https://github.com/Dolibarr/dolibarr/pull/27528/commits/c2345a48c3a915badb7675ca363e1cab6b271328